src/Microsoft.Azure.SignalR.Common/Endpoints/ServiceEndpointManagerBase.cs

```diff
@@ -230,6 +230,9 @@ private async Task RemoveHubServiceEndpointAsync(HubServiceEndpoint endpoint, Ca
 
             // Set complete
             endpoint.CompleteScale();
+
+            // Dispose endpoint
```

Comment:
As endpoints are shared between hubs, will it matter when an endpoint is disposed in one hub, while requests come in another hub?

Reply:
How about sharing a global timer between AAD access key?